--- DxLib.cpp	2019-03-18 02:51:49.442365704 -0400
+++ DxLib.cpp	2019-03-18 19:46:06.224809512 -0400
@@ -139,8 +139,8 @@
 	    {
 		if(event.jaxis.axis == JOYSTICK_XAXIS)
 		{
-		    if(event.jaxis.value < 0) keysHeld[SDLK_LEFT] = true;
-		    else if(event.jaxis.value > 0) keysHeld[SDLK_RIGHT] = true;
+		    if(event.jaxis.value < -1024) keysHeld[SDLK_LEFT] = true;
+		    else if(event.jaxis.value > 1024) keysHeld[SDLK_RIGHT] = true;
 		    else {
 			keysHeld[SDLK_LEFT] = false;
 			keysHeld[SDLK_RIGHT] = false;
@@ -148,8 +148,8 @@
 		}
 		else if(event.jaxis.axis == JOYSTICK_YAXIS)
 		{
-		    if(event.jaxis.value < 0) keysHeld[SDLK_UP] = true;
-		    else if(event.jaxis.value > 0) keysHeld[SDLK_DOWN] = true;
+		    if(event.jaxis.value < -1024) keysHeld[SDLK_UP] = true;
+		    else if(event.jaxis.value > 1024) keysHeld[SDLK_DOWN] = true;
 		    else {
 			keysHeld[SDLK_UP] = false;
 			keysHeld[SDLK_DOWN] = false;
